block/header_test.go

```diff
@@ -6,7 +6,7 @@
 package block
 
 import (
-	"math/rand"
+	"crypto/rand"
```

Comment:
should it also be math/v2 ?

Reply:
Nope, `math/rand/v2` does not provide `rand.Read` function